src/patternfly/components/Tile/tile.scss

```diff
@@ -25,19 +28,18 @@
   --pf-c-tile--hover__icon--Color: var(--pf-global--primary-color--100);
   --pf-c-tile--m-selected__icon--Color: var(--pf-global--primary-color--100);
   --pf-c-tile--m-disabled__icon--Color: var(--pf-global--disabled-color--100);
+  --pf-c-tile--focus__icon--Color: var(--pf-global--primary-color--100);
 
   // Stacked variatiion
   --pf-c-tile__header--m-stacked__icon--MarginBottom: var(--pf-global--spacer--xs);
   --pf-c-tile__header--m-stacked__icon--FontSize: var(--pf-global--icon--FontSize--lg);
   --pf-c-tile__header--m-stacked__icon--Color: var(--pf-global--Color--100);
   --pf-c-tile--hover__header--m-stacked__icon--Color: var(--pf-global--primary-color--100);
   --pf-c-tile--m-selected__header--m-stacked__icon--Color: var(--pf-global--primary-color--100);
-  --pf-c-tile--m-disabled__header--m-stacked__icon--Color: var(--pf-global--disabled-color--100);
```

Comment:
Looks like you need to add this back. It's used but no longer defined:

https://github.com/patternfly/patternfly/blob/baa6bcbdef670d447e3831012d1c5215025c8827/src/patternfly/components/Tile/tile.scss#L101